astro/tests/fixtures/terraform

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -euo pipefail
+readonly VERSION='Terraform v0.12.7'
+
+# When Bash receives a signal for which a trap has been set
+# while waiting for a command to complete,
+# the trap will not be executed until the command completes.
+# http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_12_02.html
+fake_work() {
+    local -r seconds="$1"
+    for ((i=0;i<=$(($seconds*10)); i++)) do
```

Comment:
Out of curiosity: is there any reason for not doing `while(true)` here?

Reply:
I made this function more flexible to be able to control the execution time